src/promptflow/promptflow/_core/token_provider.py

```diff
@@ -0,0 +1,65 @@
+import threading
+import time
+
+from abc import ABC, abstractmethod
+from azure.identity import DefaultAzureCredential
+from datetime import datetime, timedelta
+
+COGNITIVE_AUDIENCE = "https://cognitiveservices.azure.com/"
```

Comment:
why is this a constant? may document why is this value

Reply:
This is the required audience for a valid token to access azure openai service.